google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/SchemaAwareStreamWriter.java

```diff
@@ -125,6 +130,9 @@ private Message buildMessage(T item)
       return this.toProtoConverter.convertToProtoMessage(
           this.descriptor, this.tableSchema, item, ignoreUnknownFields);
     } catch (Exceptions.DataHasUnknownFieldException ex) {
+      if (this.skipRefreshStreamWriter) {
```

Comment:
false?

Reply:
I think we should have unit test for this...

Reply:
This is correct I think -- it means when refresh is skipped we directly return error
It's already covered by unit tests